packages/Modal/src/index.tsx

```diff
@@ -25,34 +25,23 @@ interface IModalProps {
 }
 const Modal: React.SFC<IModalProps> = ({
   children,
-  simple,
-  alert,
+  isSimple,
+  isAlert,
   title,
-  upload,
+  isUpload,
   actions,
-  isActive,
-  buttonText,
-  buttonModifiers,
+  overlay,
```

Comment:
I will use a default value set to `true` so you only have to specify `false` when you don't want the overlay